src/api/v2/paths/directory.js

```diff
@@ -10,6 +12,16 @@ import { getDirectories } from '../db/ldap/directory-dao';
  */
 const get = async (req, res) => {
   try {
+    const errors = [];
+    _.forEach(req.query, (value, key) => {
+      if (!value) {
+        errors.push(`Query parameter '${key}' must have a value.`);
```

Comment:
Be careful that `0` is also a [falsy value](https://stackoverflow.com/questions/19839952/all-falsey-values-in-javascript/19839953#19839953). It should be fine for now since we don't have a number type parameter at this moment, but we could possibly have in the future. So, I'll still recommend adding logic to exclude `0`.

Reply:
Also, I belive you can use [_.reduce](https://lodash.com/docs/4.17.15#reduce) and [_.pickBy](https://lodash.com/docs/4.17.15#pickBy) to improve the logic here.